polkadot/consensus/src/lib.rs

```diff
@@ -540,7 +541,13 @@ impl<C, R, P> bft::Proposer<Block> for Proposer<C, R, P>
 			next_index += 1;
 
 			let signature = MaybeUnsigned(self.local_key.sign(&extrinsic.encode()).into());
-			let uxt = UncheckedExtrinsic { extrinsic, signature };
+
+			let extrinsic = Extrinsic {
+				signed: extrinsic.signed.into(),
```

Comment:
just double-checking -- it is fine to have the `Address` be the same as the `AccountId` and forgo the lookup?

Reply:
`AccountId::into()` results in an `Address::Id` that represents the former.